hknweb/exams/templates/exams/index.html

```diff
@@ -21,12 +34,21 @@
 
 {% block content %}
 <div id="actual-contents">
+    <center>
+        <select id="selectSemestercourses" style="width: 300px">
+            <option selected disabled="true">--- Search for Course ---</option>
+            {% for coursesemester in searchCourses %}
+            <option value="/exams/course/{{ coursesemester.course.department.abbreviated_name }}/{{ coursesemester.course.number }}/?term={{ coursesemester.semester.semester }}">{{coursesemester.semester.semester}} {{coursesemester.course.department.abbreviated_name}} {{coursesemester.course.number}}</option>
+            {% endfor %}
+        </select>
+    </center>
+    <br>
     Browse by department:
```

Comment:
This looks kinda awkward since the department headings are so large; let's remove it?
![image](https://user-images.githubusercontent.com/46059916/91670429-3fa48600-ead2-11ea-8983-0e9815791c9f.png)